app/_src/gateway-operator/get-started/kic/create-gateway.md

```diff
@@ -156,7 +156,7 @@ spec:
         spec:
           containers:
           - name: proxy
-            image: kong:{{ site.data.kong_latest_gateway.ce-version }}
+            image: kong/kong:{{ site.data.kong_latest_gateway.ce-version }}
```

Comment:
This isn't a typo. `kong` is the official image.

Reply:
Looks like the user had issues pulling it this way: https://github.com/Kong/docs.konghq.com/issues/8304

I can close this if it's due to environment problems instead.

Reply:
I think it's due to the environment